NullPointerException with symbols

I have not studied the problem, but I did a stupid copy/paste of the following content in the` microprofile-config.properties` editor:

```
Caused by: java.lang.NullPointerException
    at org.eclipse.lsp4mp.model.Property.getResolvedPropertyValue(Property.java:181)
    at org.eclipse.lsp4mp.model.PropertyValueExpression.getResolvedValue(PropertyValueExpression.java:68)
    at org.eclipse.lsp4mp.services.MicroProfileHover.getPropertyValueExpressionHover(MicroProfileHover.java:221)
    at org.eclipse.lsp4mp.services.MicroProfileHover.doHover(MicroProfileHover.java:84)
    at org.eclipse.lsp4mp.services.MicroProfileLanguageService.doHover(MicroProfileLanguageService.java:106)
    at org.eclipse.lsp4mp.ls.ApplicationPropertiesTextDocumentService.lambda$hover$3(ApplicationPropertiesTextDocumentService.java:187)
    at java.base/java.util.concurrent.CompletableFuture.biApply(CompletableFuture.java:1236)
    ... 7 more
```

and outline is empty and we have a NullPointerException:

```
Caused by: java.lang.NullPointerException
	at org.eclipse.lsp4mp.services.MicroProfileSymbolsProvider.findDocumentSymbols(MicroProfileSymbolsProvider.java:93)
	at org.eclipse.lsp4mp.services.MicroProfileLanguageService.findDocumentSymbols(MicroProfileLanguageService.java:129)
	at org.eclipse.lsp4mp.ls.ApplicationPropertiesTextDocumentService.lambda$documentSymbol$7(ApplicationPropertiesTextDocumentService.java:198)
	at java.base/java.util.concurrent.CompletableFuture.biApply(CompletableFuture.java:1236)
	... 7 more
```

--- a/microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileSymbolsProvider.java
+++ b/microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileSymbolsProvider.java
@@ -90,10 +90,12 @@ class MicroProfileSymbolsProvider {
 					for (String path : paths) {
 						symbol = getSymbol(path, property, symbol != null ? symbol.getChildren() : symbols);
 					}
-					symbol.setKind(SymbolKind.Property);
-					String value = property.getPropertyValue();
-					if (value != null) {
-						symbol.setDetail(value);
+					if (symbol != null) {
+						symbol.setKind(SymbolKind.Property);
+						String value = property.getPropertyValue();
+						if (value != null) {
+							symbol.setDetail(value);
+						}
 					}
 				}
 			}

--- a/microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/ApplicationPropertiesSymbolsTest.java
+++ b/microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/ApplicationPropertiesSymbolsTest.java
@@ -23,7 +23,7 @@ import org.junit.Test;
 
 /**
  * Test with symbols in 'application.properties' file.
- * 
+ *
  * @author Angelo ZERR
  *
  */
@@ -103,4 +103,17 @@ public class ApplicationPropertiesSymbolsTest {
 				s("quarkus.application.name", SymbolKind.Property, "application.properties", r(0, 0, 3, 4))
 		);
 	};
+
+	@Test
+	public void justPeriod() throws BadLocationException {
+		String value = ".";
+		testDocumentSymbolsFor(value);
+	}
+
+	@Test
+	public void justEquals() throws BadLocationException {
+		String value = "=";
+		testDocumentSymbolsFor(value);
+	}
+
 }

